### ISSUE TITLE
Accessibility problems on messenger.com website

### ISSUE THREAD

**Issue by lallerke1** [nvaccess/nvda#8811]
Steps to reproduce:
1. Open web browser, in this case google chrome
2. go on www.messenger.com
3. Select from the dropdown menu the active contact list.
4. By pressing tab, or arrow keys, nvda does not announce the names of the people, only the the element position of the current partner, for example 2 of 14.

Actual behaviour:
NVDA does not read the list properly.

Expected behaviour:
NVDA shall read the names of the contacts by pressing tab, or arrow keys.

System configuration:
NVDA installed

Nvda version:
alpha-16123,142e542a

Windows version:
windows 10 version 1803 build 17134.320

Name and version of other software in use when reproducing the issue:
google chrome version 69.0.3497.100

Other information about my system:
display language hungarian, but also tested in german and english.

Other questions:
As seen from the attached log file, nvda can't catch the attribute, and dom information of the list.

Yes, the issue exists after restart.
I tried the latest stable 2018.2.3 NVDA version.

[nvda.log](https://github.com/nvaccess/nvda/files/2433760/nvda.log)

**Comment by afik1231** [comment]
hi, i tried it itself now and as you just wrote, its does not work.

**Comment by Adriani90** [comment]
@lallerke1, @afik1231 can you still reproduce this issue in NVDA 2019.3 Beta 1?

**Comment by Adriani90** [comment]
Tested in NVDA 2020.1, the names are reported correctly, they are having the link atribute. This works both in focus and browse mode. Closing as works for me.
